libraries/libvapours/include/vapours/results/results_common.hpp

```diff
@@ -177,7 +177,10 @@ namespace ams {
                 static_assert(Value != Base::SuccessValue, "Value != Base::SuccessValue");
             public:
                 constexpr ALWAYS_INLINE operator Result() const { return MakeResult(Value); }
-                constexpr ALWAYS_INLINE operator ResultSuccess() const { OnResultAbort(Value); }
+                constexpr ALWAYS_INLINE operator ResultSuccess() const {
+                    OnResultAbort(Value);
+                    return ResultSuccess();
```

Comment:
I am very uncomfortable with this. OnResultAbort(Value) is NORETURN... Can __builtin_unreachable() or AMS_ASSUME(false) be used instead?

Reply:
Added __builtin_unreachable() before the return. Unfortunately Clang requires a return statement to be present here even if the end of the function can never be reached...